libobs/obs.h

```diff
@@ -291,6 +297,16 @@ EXPORT uint32_t obs_get_version(void);
 /** @return The current core version string */
 EXPORT const char *obs_get_version_string(void);
 
+/** Sets things up for calls to obs_get_cmdline_args. Called once at startup. */
+EXPORT void obs_set_cmdline_args(int argc, char **argv);
+
+/**
+ * Get the argc/argv used to start OBS
+ * @returns A obs_cmdline_args struct with the argc/argv used to start OBS.
```

Comment:
`@returns` should show the return type separately, and then a description of what is returned.

Reply:
All right, will update.